bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/link/AbstractLink.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.thing.link;
+
+import org.eclipse.smarthome.core.thing.UID;
+
+/**
+ * {@link AbstractLink} is the abstract base class of all links.
+ * 
+ * @author Dennis Nobel - Initial contribution
+ */
+public abstract class AbstractLink {
+
+    /**
+     * Returns the link ID for a given item name and UID
+     * 
+     * @param itemName
+     *            item name
+     * @param uid
+     *            UID
+     * @return the item channel link ID
+     */
+    public static String getIDFor(String itemName, UID uid) {
+        return itemName + " -> " + uid.toString();
```

Comment:
Can not be fixed easily, because ID is already used as key in persistent storages.